libcloudforensics/providers/gcp/internal/project.py

```diff
@@ -60,10 +55,11 @@ def __init__(self,
     self._function = None
     self._build = None
     self._log = None
+    self._storage = None
     self._monitoring = None
 
   @property
-  def compute(self) -> compute_module.GoogleCloudCompute:
+  def compute(self) -> 'compute_module.GoogleCloudCompute':
```

Comment:
Why add the quotes `''` here for the return types? It seems that mypy was happy without them before (afaik you only need the quotes when the type has not be defined yet (see https://www.python.org/dev/peps/pep-0484/#forward-references)) 

Did something change in the way things are imported?

Same for following methods